ui/apps/platform/cypress/integration/vulnerabilities/workloadCves/imageSingle.test.js

```diff
@@ -0,0 +1,175 @@
+import capitalize from 'lodash/capitalize';
+
+import withAuth from '../../../helpers/basicAuth';
+import {
+    applyLocalSeverityFilters,
+    selectResourceFilterType,
+    typeAndEnterResourceFilterValue,
+    visitWorkloadCveOverview,
+} from './WorkloadCves.helpers';
+import { selectors } from './WorkloadCves.selectors';
+
+describe('Image Single page', () => {
+    withAuth();
+
+    it('should correctly handle Image single page specific behavior', () => {
+        visitWorkloadCveOverview();
+
+        // View the 'Image' tab on the overview page and click any image link
+        cy.get(selectors.entityTypeToggleItem('Image')).click();
+        cy.get('tbody tr td[data-label="Image"] a').first().click();
+
+        // Check that only applicable resource menu items are present in the toolbar
+        cy.get(selectors.resourceDropdown).click();
+        cy.get(selectors.resourceMenuItem('CVE'));
+        cy.get(selectors.resourceMenuItem('Image')).should('not.exist');
+        cy.get(selectors.resourceMenuItem('Deployment')).should('not.exist');
+        cy.get(selectors.resourceMenuItem('Cluster')).should('not.exist');
+        cy.get(selectors.resourceMenuItem('Namespace')).should('not.exist');
+        cy.get(selectors.resourceDropdown).click();
+
+        // Check that the CVEs by severity totals in the card match the number in the "results found" text
```

Comment:
What do you think about separating into 2 tests:
1. preceding assertions are independent of data
2. following sets of assertions are dependent on data (keep together to ensure that same page is tested)

Although computation looks reliable, separation would allow a temporary `skip` in worst case if we discovered later that it failed for edge case of no results.
![No_results_found](https://github.com/stackrox/stackrox/assets/11862657/b642ff78-c117-4736-8200-f0111b83e056)